core/src/util/builders.rs

```diff
@@ -235,7 +235,10 @@ impl ImportBundleBuilder {
 		};
 
 		ImportBundle {
-			dsnp_keys: DsnpKeys { keys, keys_hash, dsnp_user_id: self.dsnp_user_id },
+			dsnp_keys: match keys.len() {
+				0 => None,
+				_ => Some(DsnpKeys { keys, keys_hash, dsnp_user_id: self.dsnp_user_id }),
+			},
```

Comment:
This probably needs to be done in `ImportBundleBuilder.ts`

Reply:
In `import-bundle-builder.ts`, it simply allows `dsnpKeys` to be undefined (previously was inserting a dummy default object if undefined)